### PR TITLE
Added traefik.path.strip, traefik.path.prefix, traefil.path.prefix.strip

### PR DESCRIPTION
**Pull request by bmanth60**
Hi,

RE: https://github.com/rawmind0/rancher-traefik/issues/23

I have added support for  traefik.path.strip, traefik.path.prefix, and traefik.path.prefix.strip. I have ran the docker build command and it has built successfully. There are no testing instructions so I have not tested this.

Also, I have bumped up the version, is that correct?

Thanks! Please review!

### CODE REVIEW

**Comment by ls-amine** [comment]
@bmanth60 great job!

File: root/opt/tools/confd/etc/templates/rules.toml.tmpl [hunk 1]
```diff
@@ -73,16 +73,49 @@
...
+        {{- if exists (printf "/stacks/%s/services/%s/labels/traefik.path.prefix" $stack_name $service_name) -}}
+          PathPrefix:
+          {{- $paths := replace (getv (printf "/stacks/%s/services/%s/labels/traefik.path.prefix" $stack_name $service_name)) " " "" -1 -}}
+          {{- $path := split $paths "," -}}
+          {{- range $i, $p := $path -}}
+            {{- if $i -}},{{- end -}}
+            {{- toLower $p -}}
+          {{- end -}}
+          ;
+        {{- end}}
```
Comment (ls-amine) [inline root/opt/tools/confd/etc/templates/rules.toml.tmpl]: should be `{{- end -}}`

File: root/opt/tools/confd/etc/templates/rules.toml.tmpl [hunk 2]
```diff
@@ -73,16 +73,49 @@
...
+        {{- if exists (printf "/stacks/%s/services/%s/labels/traefik.path.prefix.strip" $stack_name $service_name) -}}
+          PathPrefixStrip:
+          {{- $paths := replace (getv (printf "/stacks/%s/services/%s/labels/traefik.path.prefix.strip" $stack_name $service_name)) " " "" -1 -}}
+          {{- $path := split $paths "," -}}
+          {{- range $i, $p := $path -}}
+            {{- if $i -}},{{- end -}}
+            {{- toLower $p -}}
+          {{- end -}}
+          ;
+        {{- end}}
```
Comment (ls-amine) [inline root/opt/tools/confd/etc/templates/rules.toml.tmpl]: should be `{{- end -}}`

**Review by ls-amine — COMMENTED** [review]
I did test the changes, had issues.. comments below , regarding how to fix it.
also in order to support priority. code below works well : 
default to 5 if label is not setup.
to be added under `passHostHeader`
```
priority =  {{ if exists (printf "/stacks/%s/services/%s/labels/traefik.priority" $stack_name $service_name) -}}
                {{  getv (printf "/stacks/%s/services/%s/labels/traefik.priority" $stack_name $service_name)  }}
                {{ else -}}
                  5
                {{- end}}
```

**Comment by bmanth60** [comment]
@ls-amine Thanks for the feedback! I have added the priority code and added the missing dashes to the config sections.

**Comment by rawmind0** [comment]
Hi guys,

thanks so much for the PR and the review. Nice job!!! :D

LGTM

**Pull request merged**